packages/migrate/src/__tests__/DbPull/postgresql-multischema.test.ts

```diff
@@ -88,7 +88,7 @@ describe('postgresql-multischema', () => {
       "Prisma schema validation - (get-config wasm)
       Error code: P1012
       error: If provided, the schemas array can not be empty.
-        -->  schema.prisma:4
+        -->  with-schemas-in-datasource-0-value.prisma:4
```

Comment:
Similar to Alberto's PR for CLI before - this is actually a fix, `schema.prisma` was a hardcoded name before, now we use actual file names.